Sources/DarkModeCore/UIColor+DarkModeKit.h

```diff
@@ -16,11 +18,18 @@ NS_ASSUME_NONNULL_BEGIN
 
 + (UIColor *)dm_colorWithLightColor:(UIColor *)lightColor darkColor:(UIColor *)darkColor
 NS_SWIFT_UNAVAILABLE("Use init(_:light:dark:) instead.");
++ (UIColor *)dm_colorWithDynamicProvider:(UIColor * (^)(DMTraitCollection *traitCollection))dynamicProvider
+NS_SWIFT_UNAVAILABLE("Use init(_:dynamicProvider:) instead.");
+- (UIColor *)dm_resolvedColorWithTraitCollection:(DMTraitCollection *)traitCollection
```

Comment:
I'm not sure if we should add `dm_resolvedColorWithTraitCollection` to UIColor extension, since the lightColor, darkColor properties were added to DynamicColor instead of UIColor extension

Reply:
Depending on whether end users will directly need this API.

Reply:
Sounds good.